pkgs/applications/networking/protonmail-bridge/default.nix

```diff
@@ -2,34 +2,43 @@
 
 buildGoModule rec {
   pname = "protonmail-bridge";
-  version = "1.5.0";
+  version = "1.6.6";
 
   src = fetchFromGitHub {
     owner = "ProtonMail";
     repo = "proton-bridge";
     rev = "br-${version}";
-    sha256 = "lHqwKlFwz9iO7LJMGFTGCauw12z/BKnQte2sVoVkOaY=";
+    sha256 = "1qy7308k4c33g3agxc8hjc9bhrqr7cqsna9pgpjzay64mhamgdrc";
   };
 
-  vendorSha256 = "eP+7fqBctX9XLCoHVJDI/qaa5tocgg3F5nfUM6dzNRg=";
+  vendorSha256 = "04aa7syp5hhpqxdpqlsmmbwywnbrh4ia0diym2935jbrqccnvm1k";
 
   nativeBuildInputs = [ pkg-config ];
 
   buildInputs = [ libsecret ];
 
   buildPhase = ''
+    runHook preBuild
+
+    patchShebangs ./utils/
     make BUILD_TIME= build-nogui
+
+    runHook postBuild
   '';
```

Comment:
```suggestion
  preBuild = ''
    patchShebangs ./utils/
  '';
  
  makeFlags = [
    "BUILD_TIME="
    "build-nogui"
  ];
```

Reply:
I don't think makeFlags work with buildGoModule. I'm curious to find out.

Reply:
It seems to me that `makeFlags` is ignored by `buildGoModule`.

Reply:
Oh, then we need to add ``-j$NIX_BUILD_CORES``.

Reply:
Done, thank you for your feedback.